src/adagrams.js

```diff
@@ -1,8 +1,120 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const tilesAmount = {
+      A: 9,
+      B: 2,
+      C: 2,
+      D: 4,
+      E: 12,
+      F: 2,
+      G: 3,
+      H: 2,
+      I: 9,
+      J: 1,
+      K: 1,
+      L: 4,
+      M: 2,
+      N: 6,
+      O: 8,
+      P: 2,
+      Q: 1,
+      R: 6,
+      S: 4,
+      T: 6,
+      U: 4,
+      V: 2,
+      W: 2,
+      X: 1,
+      Y: 2,
+      Z: 2
+    };
+
+
+    let tiles = [];
+    for (let [key, value] of Object.entries(tilesAmount)) {
+      tiles.push(key.repeat(value).split(''));
+    }
+    tiles = tiles.flat()
+    const numberOfTiles = 10;
+    //https://stackoverflow.com/questions/19269545/how-to-get-n-no-elements-randomly-from-an-array
+    const shuffled = tiles.sort(() => 0.5 - Math.random());
+    let hand = shuffled.slice(0, numberOfTiles);
```

Comment:
This is minor: There are a few places like this where you don't have a need to reassign the variable but you aren't declaring it with const. 
It's usually preferred to declare variables with const unless you have a need to reassign them.